Source/WebCore/Modules/mediasession/MediaSession.cpp

```diff
@@ -41,14 +41,15 @@
 #include "Navigator.h"
 #include "Page.h"
 #include "PlatformMediaSessionManager.h"
+#include <wtf/CryptographicallyRandomNumber.h>
 #include <wtf/JSONValues.h>
 #include <wtf/SortedArrayMap.h>
 
 namespace WebCore {
 
 static const void* nextLogIdentifier()
 {
-    static uint64_t logIdentifier = cryptographicallyRandomNumber();
+    static uint64_t logIdentifier = cryptographicallyRandomNumber<uint32_t>();
```

Comment:
Left this as is, generating a 32-bit random number and storing it in a 64-bit global. New function name makes that a lot more obvious. Not sure if it’s a mistake.